built_in_tasks/target_capture_task.py

```diff
@@ -401,7 +401,7 @@ def _start_target(self):
         super()._start_target()
 
         # Show target if it is hidden (this is the first target, or previous state was a penalty)
-        target = self.targets[self.target_index % 2]
+        target = self.targets[self.target_index % self.chain_length]
```

Comment:
can you revert the changes to this task? i think this will break random target chains.